app/assets/javascripts/hyrax/app.js

```diff
@@ -40,6 +41,14 @@ Hyrax = {
       var controls = new CollectionControls($('#collection-controls'));
     },
 
+    // Collection types
+    collectionTypes: function() {
+      var element = $('.collection-types-wrapper')
+      if (element.length > 0) {
+        var CollectionTypes = require('hyrax/collection_types');
```

Comment:
@jeremyf Yea agree.  I followed the lead of how the 'workEditor' function (line 60 in app.js) created a new Editor instance.  I can move this logic into the actual class itself though.  Would that work?